docs/reference/docs/termvectors.asciidoc

```diff
@@ -86,6 +86,40 @@ or the field statistics of the entire index, and not just at the shard. Use it
 with caution as distributed frequencies can have a serious performance impact.
 
 [float]
+==== Terms Filtering coming[2.0]
+
+With the parameter `filter`, the terms returned could also be filtered based
```

Comment:
I'm nervous about using the word `filter` here, as we try to reserve that purely for query DSL filters (although we do use it for fielddata filters as well http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/fielddata-formats.html#field-data-filtering). I'm struggling to come up with a better name though.

I'm wondering if we need this sub-level at all, or if we can just move the `max_num_terms` etc parameters up to the top level?  There doesn't seem to be any name clash (although I do like the fact that having them as sub-params indicates their action more clearly).